orchagent/portsorch.cpp

```diff
@@ -2823,6 +2823,12 @@ bool PortsOrch::addLag(string lag_alias)
     lag.m_members = set<string>();
     m_portList[lag_alias] = lag;
 
+    /* Add lag name map to counter table */
```

Comment:
It is essentially adding Lag to `COUNTERS_PORT_NAME_MAP`. Currently it contains only the physical interfaces. I don't think we want to overload this table with LAG alias. Suggest to create a new table `COUNTERS_LAG_NAME_MAP `and you can modify the function https://github.com/Azure/sonic-py-swsssdk/blob/master/src/swsssdk/port_util.py#L46 to fetch the interfaces for the `fdbshow`. @stcheng what do you think?

Reply:
agree with @prsunny

Reply:
Thanks @stcheng and @prsunny. I will make the changes and resubmit.

Reply:
@anilkpandey , can you resubmit the changes based on above feedback?